tap-snapshots/test/arborist/reify.js.test.cjs

```diff
@@ -695,6 +696,7 @@ ArboristNode {
           "type": "prod",
         },
       },
+      "hasShrinkwrap": true,
```

Comment:
curious what this change is about.. were we missing a path somewhere that was supposed to be populating this before? same for the inShrinkwrap above (and both got added in a bunch of places)

Reply:
There's a second commit in the PR that adds the shrinkwrap/bundle info to the pretty-printed nodes.  It was a huge pain to track this issue down without adding that, so I landed it as well.

Reply:
:+1: makes sense, and definitely helpful for debugging